pkg/provider/resource_cluster.go

```diff
@@ -379,14 +396,22 @@ func (c *clusterResource) Create(ctx context.Context, req resource.CreateRequest
 		return
 	}
 
-	clusterId, err := c.client.Create(ctx, config.ProjectId, makeClusterForCreate(config))
+	cluster, err := c.makeClusterForCreate(ctx, config)
```

Comment:
A suggestion:
```suggestion
	clusterModel, err := c.makeClusterForCreate(ctx, config)
```
For increasing readability? WDYT?
The same would also apply for the method calls in the Update part.

Reply:
sure